<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSellingPoint</class>
 <widget class="QDialog" name="frmSellingPoint">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selling point calculation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="text">
        <string>Selling point calculation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select your gain</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radTPC">
            <property name="text">
             <string>Select a gain percentage</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbTPC">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>2.5 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7.5 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 %</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radGain">
            <property name="text">
             <string>Select a gain amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="myQLineEdit" name="txtGanancia">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select a gain amount</string>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="radPrice">
            <property name="text">
             <string>Select a price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="myQLineEdit" name="txtPrice">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>10</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Selling point calculation options</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="chkGainsTime">
            <property name="text">
             <string>Remove operations less than a year</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPonderanAll">
            <property name="text">
             <string>Include shares of the same product from different investments</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_current">
        <attribute name="title">
         <string>Current state</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="myQTableWidget" name="table">
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Investment (Bank)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Operation type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Shares</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Bought price</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Invested</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pending</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Selling point state</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="myQTableWidget" name="tableSP">
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Investment (Bank)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Operation type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Shares</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Price</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Invested</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pending</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Set the selling point to 0 â‚¬</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>txtGanancia</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radGain</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPrice</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radPrice</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbTPC</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>radTPC</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
